R/zchunk_L254.transportation_UCD.R

```diff
@@ -78,16 +79,22 @@ module_energy_L254.transportation_UCD_DISABLED <- function(command, ...) {
 
     # Load required inputs
     GCAM_region_names <- get_data(all_data, "common/GCAM_region_names")
-    UCD_techs <- get_data(all_data, "energy/UCD_techs")
+    UCD_techs <- get_data(all_data, "energy/mappings/UCD_techs")
     A54.demand <- get_data(all_data, "energy/A54.demand")
-    A54.demand_ssp1 <- get_data(all_data, "energy/A54.demand_ssp1")
-    A54.incelas_R <- get_data(all_data, "energy/A54.incelas_R")
+    # The below variable (energy.TRN_SSP) controls which scenario to run, as only one scenario can be run at a time.
+    # This is a special case, and the way this is executed will likely change in the future.
+    # Note that the variable is defined in constants.R
```

Comment:
Nicely commented--thanks.

Reply:
This still confused me, until i asked @kvcalvin what the issue was.  I think it means that just some parameters were specific to SSP1, but you would still have to run the data system completely over to get a set of transport xml files for each of SSP2-5.  Is that right, @kvcalvin ?.  Can you add comments clarifying to this file?  Thanks

Reply:
@mwisepnnl from what I understand, you can run the model under either one of two scenarios (`CORE` and `SSP1`). I say this because there are two input files for `A54.demand` and `A54.tranSubsector_VOTT`, which is what the variable controls. And the way the code is set up, you would control it through the R.constants file. For other SSPs (is `CORE` one of these?), you would need other input files that are not here. If that's right, I can expand on the comment. Please let me know

Reply:
@aarony2j you should be able to run the system using any of the SSPs. SSP1 is the only one that has a separate assumption file.